src/main/java/com/agricraft/agricore/plant/AgriPlantOld.java

```diff
@@ -0,0 +1,272 @@
+/*
```

Comment:
I suppose we should look into some sort of versioning schema for the models here. Like naming the files v1, v2, etc. I'll have to look into GSON and see what options we have for versioning.

Reply:
I agree, but for now the most important is porting the jsons to be compliant with 1.16.4 code first.
We could define a transformer to convert 1.12.2 jsons to 1.16.4, for which this will work perfectly fine.